tpl/inc/metabox.php

```diff
@@ -18,7 +18,13 @@
 		echo '</div>';
 	} else {
 		echo '<div style="display:flex;margin-bottom:10px;align-items: center;gap: 2ch;justify-content: space-between;"><label for="' . $k . '">' . $v . '</label>';
-		echo '<input class="litespeed-tiny-toggle" id="' . Str::trim_quotes($k) . '" name="' . Str::trim_quotes($k) . '" type="checkbox" value="1" ' . ($existing_val ? 'checked' : '') . ' />';
+		echo '<input class="litespeed-tiny-toggle" id="' .
+			Str::trim_quotes($k) .
+			'" name="' .
+			Str::trim_quotes($k) .
+			'" type="checkbox" value="1" ' .
+			($existing_val ? 'checked' : '') .
+			' />';
```

Comment:
With a single class you could say
```php
$input_attrs = [
    'class' => 'litespeed-tiny-toggle',
    'id' => Str::trim_quotes($k),
    'name' => Str::trim_quotes($k),
    'type' => 'checkbox',
    'value' => '1',
];
if ($existing_val) {
    $input_attrs['checked'] = null;
}
echo (new Element('input', $input_attrs))->render();
```
https://github.com/szepeviktor/SentencePress/blob/master/src/Html/Element.php 👀

Reply:
That is a good idea, just a bit overkill in current context maybe? 😌

Reply:
True. A sprintf could save you from that 😆

Reply:
I've added a new method only for you.
https://github.com/szepeviktor/SentencePress/commit/4845a8168529c1a0d8370c3186dc2b34f777bddc
so you can
```php
$input = (new Element('input', $input_attrs))->setBooleanWhen('checked', $existing_val);
echo $input->render();
```